packages/ceramic-docid/src/doctype-table.json

```
@@ -0,0 +1,4 @@
+{
+  "tile": 0,
+  "caip10-link": 1
```

Should we use hex here for consistency with spec?
```suggestion
  "tile": 0x00,
  "caip10-link": 0x01
```

---

i don't see why they are not just represented as ints (maybe to represent size as well)? most of ipfs libs use int tables internally instead of the hex, and if going to allow subnets/doctypes by code, not only strings in table, then seems ints would be nicer, not much preference though

---

ok, that makes sense. We should update the CIPs as well I guess to reflect.